packages/@ember/-internals/metal/lib/destroy.ts

```diff
@@ -0,0 +1,19 @@
+import { Meta, peekMeta } from '@ember/-internals/meta/lib/meta';
+import { schedule } from '@ember/runloop';
+import { destroyObservers } from './observer';
+
+export function destroy(obj: object): boolean {
```

Comment:
We should update `Ember.destroy` to use this. After reading, I was confused by the return value. Maybe an inline comment that explains what the boolean is for (`false` for "already done" and `true` for "just did the destroy").

Reply:
added documentation @returns